### ISSUE TITLE
Bad naming

### ISSUE THREAD

**Issue by I-Hansson** [Simbpa/TDA553-lab1#16]
The class Model acts like a controller so it should have another name.

**Comment by SkepparAbbe** [comment]
Sorry, but could you elaborate? We don't see how it acts as a controller since that's the part handling input to the model, controlling it. Do you mean that the the whole Cars module is the model or what does the Model class controll? Thank you!

**Comment by I-Hansson** [comment]
Yes the whole Cars module is the model in our program. The class that you have called **Model** is a aport of the controller.
